nemo/collections/nlp/models/token_classification/ner_model.py

```diff
@@ -54,36 +54,40 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             data_dir=cfg.data_dir, modes=["train", "test", "dev"], pad_label=cfg.pad_label
         )
         self.data_dir = cfg.data_dir
-        self.model_cfg = cfg
+        # self.model_cfg = cfg
 
         self.tokenizer = get_tokenizer(
             tokenizer_name=cfg.language_model.tokenizer,
             pretrained_model_name=cfg.language_model.pretrained_model_name,
-            vocab_file=cfg.language_model.vocab_file,
-            tokenizer_model=cfg.language_model.tokenizer_model,
+            vocab_file=self.register_artifact(
+                conf_path='language_model.vocab_file', src=cfg.language_model.vocab_file
+            ),
```

Comment:
@okuchaiev In most cases, we don't provide a vocab file and use the default one pulled by means of the "pretrained_model_name". So, in this setup vocab.txt should be provided separately for Jarvis purposes?

Reply:
I think if you are not providing vocabulary file when you create your model, then .save_to() should not include any vocabulary files inside .nemo file.

It seems like in this case you are pulling tokenizer directly from HF based on name, then, I think, Jarvis should do exactly the same. @ryanleary does it make sense?

This functionality is generic for the cases when vocabulary and/or model files are indeed specified